### ISSUE TITLE
Opening an existing sketch should close Untitled Sketch

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#157]
_From [dirk.van...@gmail.com](https://code.google.com/u/102845219688896911239/) on October 07, 2008 15:18:09_

What steps will reproduce the problem? 1. Start Fritzing
2. Leave the Untitled Sketch.fz unmodified
3. Open a previously saved sketch What is the expected output? What do you see instead? It should close the untouched Untitled document (and not cascade). Now the Untitled Sketch 
document is still open (it is currently hidden behind it ... another issue), but it is still open and 
you see if when you close the opened document or move its window.

When opening a file Fritzing should be checking if there is an Untitled Sketch.fz document which 
the user obviously doesn't want to work with, so it can be closed.
This check should be done before it opens the new window in a cascaded manner ( issue #156 ) 
(or it should later cascade it back, which probably looks not so nice because of the flicker).

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=157_
